src/features/student/views/StudentSkillView/components/StudentSkillViewContainer/StudentSkillViewContainer.vue

```diff
@@ -76,6 +76,13 @@ const displayedSection = computed<Component>(() => {
     default: return StudentSkillDetailedSection
   }
 })
+
+const sectionProps = computed<Record<string, any>>(() => {
```

Comment:
@Nathan-Pignon il faut pas utiliser any
```suggestion
const sectionProps = computed<Record<string, string>>(() => {
```

Reply:
C'est pour le cas où la props que l'on passera ne sera pas une string, de base j'étais parti sur l'objet entier mais vu que pour le moment on affiche seulement la titre je me suis dit qu'on allait passer seulement la donnée que l'on a besoin, mais je pense que ça sera amené à changer. 
Ducoup tu veux quand même que je passe en string et on fera le refacto au moment venu ou je laisse comme ça ?

Reply:
Met string et on fera le refacto avec le type complet quand ce sera opportun :+1: